region/maridia/inner-pink.json

```diff
@@ -6303,6 +6303,22 @@
                     ]}
                   ],
                   "note": "It's possible to get out of the sand suitless and without hjb after falling from the chute, by hugging the left side and moving quickly."
+                },
+                {
+                  "name": "Shinespark",
+                  "notable": false,
+                  "requires": [
+                    "canPlayInSand",
+                    "canSunkenTileWideWallClimb",
+                    "canMidairShinespark",
```

Comment:
Is this a midair? For me, from the ground uses 11 frames

Reply:
I was only jumping a little bit out of the sand.  Waiting to fall works too as a way around midairShinespark.